deploy/operator.yaml

```diff
@@ -36,6 +36,16 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
+  - seccomp-operator.k8s-sigs.io
+  resources:
+  - seccompprofiles
+  verbs:
+  - create
+  - patch
+  - get
+  - watch
+  - list
```

Comment:
Yep, let's add this to https://github.com/kubernetes-sigs/seccomp-operator/blob/master/deploy/base/rbac.yaml and run `make deployments` (kustomize has to be installed) to sync it up.

Reply:
done